example/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.15.0 <3.0.0"
```

Comment:
Caught by `flutter analyze`; `main.dart` uses one API that didn't appear until 2.15.0.